rsocket-transport-ktor/src/jvmMain/kotlin/io/rsocket/kotlin/transport/ktor/TcpServerTransport.kt

```diff
@@ -41,12 +41,12 @@ public class TcpServer(public val socket: ServerSocket) : ServerTransport<Job>,
     override val coroutineContext: CoroutineContext = socket.socketContext + Dispatchers.Unconfined
 
     @OptIn(ExperimentalCoroutinesApi::class)
-    override fun start(accept: suspend (Connection) -> Unit): Job = launch {
+    override fun start(accept: suspend (Connection) -> Unit): Job = launch(start = CoroutineStart.UNDISPATCHED) {
         supervisorScope {
             while (isActive) {
                 val clientSocket = socket.accept()
                 val connection = TcpConnection(clientSocket)
-                launch { accept(connection) }
+                launch(start = CoroutineStart.UNDISPATCHED) { accept(connection) }
```

Comment:
performance optimisation?

Reply:
minor, but yes + start handling of accepted sockets just after binding server.
Im thinking now, that this part could be reworked some time in future, not sure yet. TCP server transport is less stable now comparing to websocket